package io.jenkins.plugins.datastore;

public class DatastoreException extends RuntimeException {

  public DatastoreException(String msg, Throwable cause) {
    super(msg, cause);
  }

}
